lunar/distribution.yaml

```diff
@@ -1063,6 +1063,21 @@ repositories:
       url: https://github.com/ros-perception/perception_pcl.git
       version: lunar-devel
     status: maintained
+  pid:
+    doc:
+      type: git
+      url: https://bitbucket.org/AndyZe/pid.git
+      version: master
+    release:
+      tags:
+        release: release/kinetic/{package}/{version}
```

Comment:
This line also references `kinetic`.

Maybe we should extend the tests to detect this?